Template/Tag/VisualWebsiteOptimizerTag.web.js

```diff
@@ -0,0 +1,67 @@
+(function () {
+    var libLoaded = false;
+    var libAvailable = false;
+
+    function loadVwoCode(accountId)
+    {
+        if (libAvailable || typeof window._vwo_code === 'object') {
+            libAvailable = true;
+            libLoaded = true;
+            return;
+        }
+        if (!libLoaded) {
+            
+            libLoaded = true;
+        
+            window._vwo_code = (function() {
+                var account_id = accountId,
+                    settings_tolerance = 2000,
+                    library_tolerance = 2500,
+                    use_existing_jquery = false,
+                    // DO NOT EDIT BELOW THIS LINE
+                    f = false,
+                    d = document;
+                return {
+                    use_existing_jquery: function() {
+                        return use_existing_jquery;
+                    },
+                    library_tolerance: function() {
+                        return library_tolerance;
+                    },
+                    finish: function() {
+                        if (!f) {
+                            f = true;
+                            libAvailable = true;
+                        }
+                    },
+                    finished: function() {
+                        return f;
+                    },
+                    load: function(a) {
+                        var b = d.createElement('script');
+                        b.src = a;
+                        b.type = 'text/javascript';
+                        b.innerText;
+                        b.onerror = function() {
+                            window._vwo_code.finish();
+                        };
+                        d.getElementsByTagName('head')[0].appendChild(b);
+                    },
+                    init: function() {
+                        settings_timer = setTimeout('window._vwo_code.finish()', settings_tolerance);
+                        this.load('https://dev.visualwebsiteoptimizer.com/j.php?a=' + account_id + '&u=' + encodeURIComponent(d.URL) + '&r=' + Math.random());
+                        return settings_timer;
+                    }
+                };
+            }());
+            
+            window._vwo_settings_timer = window._vwo_code.init();
+        }
+    }
+
+    return function (parameters, TagManager) {
+        this.fire = function () {
+            loadVwoCode(parameters.get('accountId'));
```

Comment:
I would maybe double check an accountId was set. It is not really needed though as "NotEmpty" was specified.

Reply:
Thank you, I have updated my commit.